modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3638,6 +3639,13 @@ public boolean isDataTypeFile(String dataType) {
                 } else if ("bearer".equals(securityScheme.getScheme())) {
                     cs.isBasicBearer = true;
                     cs.bearerFormat = securityScheme.getBearerFormat();
+                } else if ("signature".equals(securityScheme.getScheme())) {
+                    // HTTP signature as defined in https://datatracker.ietf.org/doc/draft-cavage-http-signatures/
+                    // The registry of security schemes is maintained by IANA.
+                    // https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml
+                    // As of January 2020, the "signature" scheme has not been registered with IANA yet.
+                    // This scheme may have to be changed when it is officially registered with IANA.
```

Comment:
What about showing the comment as warning using LOGGER.warn?

I think we need to make it clear to users this is not official and may be subject to change in the future.

Reply:
ok, sure, doing.

Reply:
I am adding the following warning:
```
LOGGER.warn("Security scheme 'HTTP signature' is a draft IETF RFC and subject to change.");
```